foxy/distribution.yaml

```diff
@@ -5150,6 +5150,22 @@ repositories:
       url: https://github.com/open-rmf/stubborn_buddies.git
       version: foxy
     status: developed
+  swri_console:
+    doc:
+      type: git
+      url: https://github.com/swri-robotics/swri_console.git
+      version: ros2-devel
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/ros2-gbp/swri_console-release.git
+      version: 2.0.0-1
```

Comment:
```suggestion
```

There's not currently any foxy metadata in the ros2-gbp release repository for swri_console-release. If there's foxy content in a prior release repository I can help migrate it to the ros2-gbp repo but I would suggest waiting to add release info until it can be generated by a `bloom-release` automated PR.

Alternatively you can enter all of this data during a `bloom-release` automated PR and close this manual one.

Reply:
@nuclearsandwich I'm having troubles running bloom-release wit this, I think possibly because we're switching from the swri-release-gbp repo with dashing, to ros2-gbp with foxy and later.

Reply:
> I'm having troubles running bloom-release wit this, I think possibly because we're switching from the swri-release-gbp repo with dashing, to ros2-gbp with foxy and later.

I just double checked the ros2-gbp repository and noticed that the default branch was not set to `master` it's possible that this was at least part of the problem. I also noticed that this release repository also makes use of the `release_repo_url` track configuration field and that could be part of the problem as well.

It looks like the same issue as https://github.com/swri-robotics/marti_messages/issues/112#issuecomment-950020835 could be affecting this repository. Because the foxy track is not yet created it's probably being copied from a previous release and including the `release_repo_url` field pointing to the swri-robotics-gbp repository.

If you open an issue on https://github.com/ros2-gbp/swri_console-release with details from your bloom error and @-mention me I can take a closer look.